polyphemus/bin/polyphemus

```diff
@@ -8,4 +8,5 @@ require 'yaml'
 
 config = YAML.load(File.read(File.expand_path("../../config.yml",__FILE__)))
 
+Polyphemus.instance.setup_yabeda
```

Comment:
Here is an example of collecting metrics from polyphemus command invocations.  Calling the setup function prepares the metrics, which have to be predefined and configured.  Then, as you'll see below, we export the collected metrics by calling the write_job_metrics method of the application.